QuizApp/Domain/Quiz/Models/EmptyRequest.swift

```diff
@@ -0,0 +1,3 @@
+struct EmptyRequest: Encodable {
```

Comment:
Where is this used?

Reply:
Here, as I can not supply nil as encodable

`func startQuiz(with id: Int) async throws -> QuizSessionResponse {
        try await baseNetworkClient.post(
            url: QuizEndpoints.startSession(id: id).path,
            params: [:],
            headers: [HeaderField.contentType.key: HeaderValue.defaultContentType.value],
            requestBody: EmptyRequest())
    }`

Reply:
Can't you change the function so that it accepts `Encodable?`?